src/main/features/response/views/statement-of-means/other-dependants.njk

```diff
@@ -9,24 +9,24 @@
       <form method="post" novalidate
         class="analytics-submit-event-trigger"
         data-event-action="SoM: Do you support anyone else financially?"
-        data-event-label-from="doYouSupportAnyone">
+        data-event-label-from="declared">
```

Comment:
another GA label

Reply:
The same, declared is just value source.